src/igvfd/types/page.py

```diff
@@ -52,6 +52,17 @@ def unique_keys(self, properties):
         keys.setdefault('page:location', []).append(value)
         return keys
 
+    @calculated_property(
+        schema={
+            'title': 'Summary',
+            'type': 'string',
+            'notSubmittable': True,
+        }
+    )
+    def summary(self, title=None):
```

Comment:
I was wrong, title is required for Page. So title=None probably isn't needed here.

Reply:
fixed